### PR TITLE
Watchlist storaage

### LINKED ISSUE #79: Watchlist storage
As a user, I want to be able to save my watchlist, so that I can view it again after closing the application

### PR DESCRIPTION
**Pull request by wwweert123**
fix #79

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM, do you want to change to save watchlist once only on exit?

**Review by NeoMinWei — APPROVED** [review]
LGTM
maybe can add some logging and assertions

**Comment by NeoMinWei** [comment]
maybe can add logging and assertions?

File: src/main/java/seedu/financialplanner/visualisations/Visualizer.java
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/visualisations/Visualizer.java:1]: maybe can add comments on what the values mean?

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**